spiderable_server.js

```diff
@@ -4,6 +4,10 @@ var querystring = Npm.require('querystring');
 var urlParser = Npm.require('url');
 var crypto = Npm.require('crypto');
 
+var bound = Meteor.bindEnvironment(function(callback) {
```

Comment:
Sorry, you are correct. bindEnvironment is the correct approach.
I watched [this video](https://www.eventedmind.com/feed/meteor-what-is-meteor-bindenvironment) which really makes it clear, especially around 9-10 minutes.
In that video, the code looks like
```javascript
boundFunction = Meteor.bindEnvironment( function(...
```
I believe we should use the same sort of pattern, and not declare var bound at the top of the file.
Instead, we should add code inline at line 150 or so. For example, what I recommend is something like:
```javascript
    child_process.execFile(
      '/bin/bash',
      ['-c',
       ("exec phantomjs " + phantomJsArgs + " " + filename)],
      {timeout: REQUEST_TIMEOUT, maxBuffer: MAX_BUFFER},
      Meteor.bindEnvironment(function (error, stdout, stderr) {...
```
This eliminates the question about the name.  We don't need one.

Alternatively, we extract the `function(error, stdout, stderr)` as its own variable:
```
var processPhantomJsResponse = function (error, stdout, stderr) {...
};
processPhantomJsResponse = Meteor.bindEnvironment(processPhantomJsResponse);
```

This cleans the code up and reduces nesting.